scss/tech.scss

```diff
@@ -0,0 +1,56 @@
+.techs {
+  display: block;
+  margin: 0 auto;
+  max-width: 1000px;
+
+  .tech {
+    display: inline-block;
+    padding: 10px;
+    vertical-align: top;
+    width: 24%;
+
+    .card {
```

Comment:
This should be added to the tech class instead of creating a new one and use a single div instead of nesting 2 divs